### DISCUSSION TITLE
Tranferring Code.Org projects to GitHub

### DISCUSSION THREAD

**Discussion by Ombucha** [community/community#24005] [New to GitHub]
<p>There is a website called <a href="https://code.org/" rel="noopener nofollow ugc">Code.Org</a>, where you can create projects using code blocks. Is there a way I could transfer the actual code to GitHub as a repository?</p>

**Comment by mpboom** [comment]
<p>Hi and welcome <a class="mention" href="/Pysics">@Pysics</a> !</p>
<p>I myself am not familiar with <a href="http://code.org" rel="noopener nofollow ugc">code.org</a>, but if it’s not too much files you’ll be able to just copy-paste the code into your local editor I guess. If that isn’t feasible: is there any way you can export the data from <a href="http://code.org" rel="noopener nofollow ugc">code.org</a> in a structured manner?</p>

**Comment by Ombucha** [comment]
<p>I am not able to export the code because it is in the form of blocks. The actual code cannot even be seen.</p>

**Comment by strafe** [comment, marked as answer]
<p>There’s no way to export code blocks like those on <a href="http://Code.org" rel="noopener nofollow ugc">Code.org</a> or Scratch in any meaningful way such that they could be uploaded to GitHub, sadly.</p>

**Comment by mpboom** [comment]
<p>Ah, like that, I see. Well if there is no actual code, no, you’ll have a really hard time using that with GutHub.</p>
<p>If however, you liked coding on <a href="http://code.org" rel="noopener nofollow ugc">code.org</a>, then why not try out following a tutorial from a programming language (try Python!) and make some actual code? You can push that to GitHub!</p>

**Comment by Ombucha** [comment]
<p>Actually, I already code games in Python. The game I made on <a href="http://code.org" rel="noopener nofollow ugc">code.org</a> is an old one. I just wanted to know if I could transfer it to GitHub.</p>

**Comment by ianzb99** [comment]
it will take a lot of work but it is definitely possible, if you go to share, then go to advanced options, then download the "export for web" file

**Comment by EunseokOhPeter** [comment]
**There is a way though**, you can click share button on the **edit screen of your project**, then click show **advanced options**, then click **export**, then download it. Html needs **code view**, so use **ctrl u**. Then you can upload those files to [**Github**](https://github.com). Although, blocks won't work though codes will work same if you run this correctly. Thanks!
